CovertInfiltration/Localization/XComGame.int

```diff
@@ -137,6 +137,22 @@ Description="<Ability:OPPORTUNE_MOMENT_1_DETECTION_MODIFIER/>% Detection Radius
 FriendlyName="Opportune Moment II"
 Description="<Ability:OPPORTUNE_MOMENT_2_DETECTION_MODIFIER/>% Detection Radius Reduction, +<Ability:OPPORTUNE_MOMENT_2_CRIT_BONUS/>% crit"
 
+[MessyInsertion X2SitRepTemplate]
+FriendlyName="Messy Insertion"
+Description="The squad begins the mission slightly tired and one soldier will be lightly injured"
```

Comment:
Define "slightly tired" - it can be interpreted differently and the variance of the "value" of these interpretations is pretty significant

Reply:
I'll remove the slightly and the lightly, I don't really want to bother putting in an XGParamTag for these descriptions to relay the values from config

Reply:
Nevermind on that I just needed to look three lines up for the answer :P